apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -145,40 +176,43 @@ Func demosaic(Func deinterleaved) {
                              c == 1, g(x, y),
                                      b(x, y));
 
-
-    /* THE SCHEDULE */
-    int vec = target.natural_vector_size(UInt(16));
-    if (target.has_feature(Target::HVX_64)) {
-        vec = 32;
-    } else if (target.has_feature(Target::HVX_128)) {
-        vec = 64;
-    }
-    g_r.compute_at(processed, yi)
-        .store_at(processed, yo)
-        .vectorize(x, vec, TailStrategy::RoundUp)
-        .fold_storage(y, 2);
-    g_b.compute_at(processed, yi)
-        .store_at(processed, yo)
-        .vectorize(x, vec, TailStrategy::RoundUp)
-        .fold_storage(y, 2);
-    output.compute_at(processed, x)
-        .vectorize(x)
-        .unroll(y)
-        .reorder(c, x, y)
-        .unroll(c);
-
-    if (target.features_any_of({Target::HVX_64, Target::HVX_128})) {
-        g_r.align_storage(x, vec);
-        g_b.align_storage(x, vec);
-    }
-
-    return output;
+    Scheduler scheduler = [=](Func processed) mutable {
```

Comment:
processed could also just be a class member, right? Though it's probably nice to have an example of this idiom in the repo to point people to.